src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -4,6 +4,7 @@
  * Add some custom fields that could be only in Truck
  * Do not remove no-field constructor
  */
+
 public class Truck extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. It's good practice to start the class body immediately after the class declaration line to avoid unnecessary whitespace.